schedule/jeos/sle/minimalvm-toolchains.yaml

```diff
@@ -0,0 +1,44 @@
+description: 'Test suite verifies functionality and integration of Development-Tools (sdk) and Desktop-Applications (desktop) patterns'
```

Comment:
Note: Copied from `schedule/jeos/sle/jeos-base+sdk+desktop.yaml` and removed the libvorbis file therein.

Reply:
We want to migrate from yamls anyway, so meh, let's leave the description.

Reply:
I just fixed it, but thanks!